src/main.rs

```diff
@@ -428,8 +428,8 @@ fn is_scheduled_jobs_disabled() -> bool {
 }
 
 #[tokio::main(flavor = "current_thread")]
-async fn main() {
-    dotenv::dotenv().ok();
+async fn main() -> anyhow::Result<()> {
+    dotenvy::dotenv()?;
```

Comment:
This means that triagebot will crash if `.env` doesn't exist. That doesn't sound right.

Reply:
Well, if we don't have it, we don't have the GitHub and Zulip token, we also don't have access to the database (it's connection conf are in it), so we can't do anything.

Reply:
Agree that for the other bins it maybe be too strict, they don't all required it.

Reply:
But it's not strictly needed, right? Someone could just configure the environment variables through other means, there are many ways to do it. I don't think that the `.env` file should be *required* to run triagebot :)

Reply:
I was under the impression that the `dotenvy` crate did more than it actually does, mainly that we were reading the values from it and not from the actual environment variables. I therefor agree that we shouldn't require a `.env`.